spot_rl_experiments/spot_rl/envs/place_env.py

```diff
@@ -219,4 +229,62 @@ def get_observations(self):
             "joint": self.get_arm_joints(self.config.SEMANTIC_PLACE_JOINT_BLACKLIST),
             "is_holding": np.ones((1,)),
         }
+        print("---")
+        return observations
+
+
+class SpotSemanticPlaceEEEnv(SpotSemanticPlaceEnv):
+    """This is Spot semantic place class"""
+
+    def __init__(self, config, spot: Spot, use_semantic_place: bool = False):
+        # We set the initial arm joints
+        config.INITIAL_ARM_JOINT_ANGLES = copy.deepcopy(
```

Comment:
don't need this anymore since we are inheriting from SpotSemanticPlaceEnv which already does this

Reply:
this part specifically:

config.INITIAL_ARM_JOINT_ANGLES = copy.deepcopy(
            config.INITIAL_ARM_JOINT_ANGLES_SEMANTIC_PLACE
        )